File: 2_User_Service/database/conn.py
import mysql.connector

db = mysql.connector.connect(
  host="localhost",
  user="root",
  password="vishv",
  database="user"
)


# preparing a cursor object
cursor = db.cursor()

# creating database
# cursor.execute("CREATE DATABASE IF NOT EXISTS USER")
# cursor.execute("USE user")
# cursor.execute("""CREATE TABLE IF NOT EXISTS USERS (
#   id INT NOT NULL AUTO_INCREMENT,
#   username VARCHAR(255) NOT NULL UNIQUE,
#   password VARCHAR(255) NOT NULL,
#   email VARCHAR(255) NOT NULL UNIQUE,
#   first_name VARCHAR(255) NOT NULL,
#   last_name VARCHAR(255) NOT NULL,
#   created_at DATETIME NOT NULL,
#   PRIMARY KEY (id)
# );""")

File: API_GATEWAY/Auth/database/conn.js
import mysql from "mysql2";

  var connection = mysql.createConnection({
    host: "mysqldb",
    port: 3306,
    user: "root",
    password: "vishv",
    database: "user",
  })

connection.connect((err) => {
  if (err) {
    console.error('Error connecting to MySQL:', err);
    return;
  }
  console.log('Connected to MySQL');
});


export default connection;
